recipe/meta.yaml

```diff
@@ -1,15 +1,15 @@
-{% set version = "0.11.3" %}
+{% set version = "0.12.1" %}
 
 package:
   name: uvicorn
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/u/uvicorn/uvicorn-{{ version }}.tar.gz
-  sha256: 6fdaf8e53bf1b2ddf0fe9ed06079b5348d7d1d87b3365fe2549e6de0d49e631c
+  sha256: a461e76406088f448f36323f5ac774d50e5a552b6ccb54e4fca8d83ef614a7c2
 
 build:
-  number: 2
+  number: 0
   script: "{{ PYTHON }} -m pip install . -vv"
   skip: True  # [py2k]
```

Comment:
this is probably no longer required?

Reply:
Yes, this can be removed, conda-forge only builds for 3.6+ nowadays.